content/overview/workflow/process.md

```diff
@@ -0,0 +1,209 @@
+
+In the real world, "pure black" does not really exist (there is always _some_ light) and there is no limit to how bright things can be (so no "pure white" either). _Scene-referred processing_ attempts to retain the physical properties of the scene for as long as possible by placing the Raw data on an unbounded linear scale and only compressing the data to the dynamic range of your display after image processing is complete.
+
+In a scene-referred workflow many common tools (tone curves and levels, for example) are no longer useful ways to manipulate the image, since they rely on now-invalid definitions of black, white and gray. Experienced users may need to learn new techniques and discard old ones, but will be rewarded with much more robust and predictable outputs.
+
+---
+Set [preferences > processing > auto-apply pixel workflow defaults](../../preferences-settings/processing.md) to "scene-referred" now. 
+
+---
+
+_Enter preferences by clicking on the gear icon in the [top panel](../user-interface/top-panel.md)_. 
+
+## white balance and color calibration
+
+Most processing software uses a traditional temperature/tint model for adjusting the white balance of an image. In darktable, the [color calibration module](../../module-reference/processing-modules/color-calibration.md) provides a much more robust and flexible approach, allowing you to explicitly define the color of the light source. This is particularly useful for scenes illuminated by artificial lighting.
+
+Please note that the [white balance](../../module-reference/processing-modules/white-balance.md) module is still enabled in this approach, but its settings should not usually be altered.
+
+---
+Set [preferences > processing > auto-apply chromatic aberration defaults](../../preferences-settings/processing.md) to "modern" now.
+
+---
+## edit in a controlled environment
+
+Image processing should be performed in a controlled environment, lit by a white light source against a background approximating mid-gray, and on a monitor that has been properly calibrated.
+
+While this may not be practical in many home editing environments you _can_ control the background colors on your monitor's display. You should set the darktable color scheme to use one of the "grey" themes and use the [color assessment mode](../../module-reference/utility-modules/darkroom/color-assessment.md) when altering tones and colors in your image. Dark themes might look good but, unless you are processing images to be viewed on a cinema screen in a darkened room, they should not be used for photo processing.
```

Comment:
maybe explain what typically happens if you don't edit your images against a suitable grey background.

Reply:
No. That's in the color assessment mode documentation. This is a "do as we say" section.